### DISCUSSION TITLE
PR 할 때 docker-compose 삭제 위험성

### DISCUSSION THREAD

**Discussion by sangwonsheep** [Kernel360/E2E2-CATCHLINE#40] [Ideas]
#38 
해당 커밋을 보면 소은님이 올리셨던 docker-compose.yaml을 제가 삭제했는데 develop 브랜치에도 같이 삭제가 되었네요.
PR 올렸을 때 docker-compose.yaml 파일을 삭제하면 안될 것 같습니다!
도커 설정 파일 잘못 올렸다고 판단되는 경우 2가지 방안이 있을 것 같습니다.
1. reset
2. 설정 파일 merge

**Comment by soeunnPark** [comment]
네 감사합니다.
PR 올릴 때도 조심하겠습니다 👁️
